utils/validationUtils.js

```diff
@@ -58,3 +58,14 @@ export const oneOfAllowedScopes = Joi.string()
 export const stringAllowedSchema = Joi.string().required();
 
 export const numberAllowedSchema = Joi.number();
+
+
+export const locationSchema = JoiBase.string().custom((val, helper) => {
+  const coordinates = val.split(',');
+  if((coordinates[1] >= -90 && coordinates[1] <= 90) && (coordinates[0] >= -180 && coordinates[0] <= 180)) {
+    return val
+  }
+  return helper.message('Invalid Coordinates');
+}).error(new Error('Invalid Coordinates.'))
+
+export const typeOfVehicleSchema = Joi.string().valid(...['AUTO','BIKE','CAR']).required().error(new Error('Invalid type of vehicle.'))
```

Comment:
Wherever we using the vehicle type `['AUTO', 'BIKE', 'CAR']`, lets bring it from a constant